app/models/course/condition/assessment.rb

```diff
@@ -0,0 +1,25 @@
+class Course::Condition::Assessment < ActiveRecord::Base
+  include ActiveSupport::NumberHelper
+
+  acts_as_condition
+  belongs_to :assessment, class_name: Course::Assessment.name, inverse_of: false
+
+  default_scope { includes(:assessment) }
+
+  def title
+    if minimum_grade_percentage
+      minimum_grade_percentage_display = number_to_percentage(minimum_grade_percentage,
+                                                              precision: 2,
+                                                              strip_insignificant_zeros: true)
+      self.class.human_attribute_name(title.title,
+                                      assessment_title: assessment.title,
+                                      minimum_grade_percentage: minimum_grade_percentage_display)
+    else
+      assessment.title
+    end
+  end
+
+  def satisfied_by?(course_user)
```

Comment:
Unused method argument - `course_user`. If it's necessary, use `_` or `_course_user` as an argument name to indicate that it won't be used. You can also write as `satisfied_by?(*)` if you want the method to accept any arguments but don't care about them.